utils/gspread_api.py

```diff
@@ -39,3 +39,42 @@ def get_previous_date_total(date):
     if not previous_week_total:
         return
     return previous_week_total.value
+
+
+def update_records(data, record_name):
```

Comment:
Better tel what records exactly in the naming as it sounds same as a record(entity)